VultisigApp/VultisigApp/Chains/Ton.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Ton.Swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 20/10/24.
+//
+
+import Foundation
+import Tss
+import WalletCore
+import BigInt
+
+enum TonHelper {
+    
+    static func getPreSignedInputData(keysignPayload: KeysignPayload) throws -> Data {
+        
+        guard keysignPayload.coin.chain.ticker == "TON" else {
+            throw HelperError.runtimeError("coin is not TON")
+        }
+        
+        guard case .Ton(let sequenceNumber, let expireAt, let bounceable) = keysignPayload.chainSpecific else {
+            throw HelperError.runtimeError("fail to get to address")
```

Comment:
change the error message to "fail to get Ton chain specific"

Reply:
Changed